sql/updates/reverse-dev.sql

```diff
@@ -1,2 +1,49 @@
 -- Remove old compression policy procedures
 DROP PROCEDURE IF EXISTS _timescaledb_internal.policy_compression_execute(INTEGER, INTEGER, ANYELEMENT, INTEGER, BOOLEAN, BOOLEAN);
+
+DO $$
+DECLARE
+    caggs_nr int;
+BEGIN
+    SELECT count(*) FROM _timescaledb_catalog.continuous_agg WHERE bucket_width < 0 INTO caggs_nr;
+    IF caggs_nr > 0 THEN
+        RAISE EXCEPTION 'Downgrade is impossible since % continuous aggregates exist which use variable buckets', caggs_nr;
```

Comment:
Since this is an experimental feature you should be able to downgrade.

Reply:
@mfundul Sure. You are able to downgrade, but first, you have to _manually_ remove the CAGGs that were created using the feature you are about to downgrade. Don't think that implicitly removing user's data is a good idea.

Reply:
You should add this as a hint.

Reply:
Done! 👍